src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java

```diff
@@ -572,8 +588,7 @@ private WebClient checkEmptyToken(String token)
     }
 
     @Override
-    public PaginationResponsePayload getAllFormDataByFormId(String formId, String relations, String filter, String sortBy, String sortOrder, Pageable pageable)
-    {
+    public PaginationResponsePayload getAllFormDataByFormId(String formId, String relations, String filter, String sortBy, String sortOrder, Pageable pageable) throws JsonProcessingException {
         PaginationResponsePayload paginationResponsePayload = new PaginationResponsePayload();
         if(elasticEnable&&relations==null)
```

Comment:
if some set elasticEnable what would happen?

Reply:
I am not able to follow if condition and the code

Reply:
If suppose they have set elasticEnable then filter should be given in elastic format as semicolon seperated values   formData.name:akhil;formData.age:18;formData.bio:softwareDeveloper